docs/resources/vulnerability_policy.md

```diff
@@ -0,0 +1,45 @@
+# Vulnerability Policy Resource
+
+The vulnerability policy resource allows for creation and management of vulnerability policies within a Cloudsmith organization.
+
+See [help.cloudsmith.io](https://help.cloudsmith.io/docs/vulnerability-policy) for the full vulnerability policies documentation.
+
+## Example Usage
+
+```hcl
+provider "cloudsmith" {
+    api_key = "my-api-key"
+}
+
+data "cloudsmith_organization" "my_organization" {
+    slug = "my-organization"
+}
+
+resource "cloudsmith_vulnerability_policy" "my_vulnerability_policy" {
+    name                    = "My Policy"
+    description             = "My vulnerability policy"
+    min_severity            = "Medium"
+    on_violation_quarantine = true
+    allow_unknown_severity  = false
+    organization            = "my-organization"
```

Comment:
NAB: It would be more idiomatic to refer to the `cloudsmith_organization` above.

Reply:
Something like this? 

`"${data.cloudsmith_organization.my_organization.slug_perm}"`